src/webparts/laserficheAdminConfiguration/LaserficheAdminConfigurationWebPart.ts

```diff
@@ -4,10 +4,8 @@ import { Version } from '@microsoft/sp-core-library';
 import {
   IPropertyPaneConfiguration,
   PropertyPaneTextField,
-  PropertyPaneDropdown,
-  IPropertyPaneDropdownOption,
   PropertyPaneToggle,
-  IPropertyPaneGroup
+  IPropertyPaneGroup,
```

Comment:
if PropertyPaneDropdown (and IPropertyPaneDropdownOption, IPropertyPaneDropdownProps
0 is not used, should delete their definition as well

Reply:
those are removed